src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -587,6 +646,24 @@ protected override bool TryComputeLength(out long length)
         }
     }
 
+    /// <summary>HttpContent that's similar to StringContent but that can be used with HTTP/2 duplex communication.</summary>
+    private sealed class StringDuplexContent : HttpContent
```

Comment:
how is it different from string conent as we are using the same default UTF8 encoding and other 2 functions seem similar as well ?